src/components/auth/LoginForm.tsx

```diff
@@ -0,0 +1,132 @@
+//auth
+import { useAuth } from './authContext';
+
+//Component
+import Banner from '../banner';
+
+//Services
+import { login, getCookie } from '../../services';
+
+//types
+import { DataUser } from '../../types/users';
+
+//CSS
+import '../../styles/login.scss';
+
+//Framework MUI
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Stack from '@mui/material/Stack';
+
+//React
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Link } from 'react-router';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+function Login() {
+  const navigate = useNavigate();
+  const [isLoading, setIsLoading] = useState(false);
+  const [serverError, setServerError] = useState<string | null>(null);
+
+  //Refresh l'auto-login
+  const { setUser } = useAuth();
```

Comment:
Je pense que si tu as un `user` depuis `useAuth` tu peux redirect vers ta page `/`
```suggestion
  const { user, setUser } = useAuth();
  if (user) {
      navigate('/');
  }
```